.yarnrc.yml

```diff
@@ -17,3 +17,8 @@ plugins:
     spec: '@yarnpkg/plugin-constraints'
 
 yarnPath: .yarn/releases/yarn-3.4.1.cjs
+
+packageExtensions:
+  '@wdio/browser-runner@*':
+    dependencies:
+      '@babel/core': '*'
```

Comment:
`@wdio/browser-runner` does not provide `@babel/core`, while it is a required peer dependency. We can solve it by manually specifying it as dependency here.

Reply:
Is there a reason to prefer this over just adding it to devDependencies where wdio is used?

Reply:
This is an issue inside `@wdio/browser-runner` and cannot be fixed by adding a dependency in the local package.